front/lib/plans/stripe.ts

```diff
@@ -0,0 +1,95 @@
+import Stripe from "stripe";
+
+import { PlanType, WorkspaceType } from "@app/types/user";
+
+const { STRIPE_SECRET_KEY = "", URL = "" } = process.env;
```

Comment:
cc @lasryaric just FYI. We do use URL in a few places in `front`. I presume we want to add it to the design doc for documentation. Should we look into renaming it to `DUST_CLIENT_FACING_URL` ?

Reply:
Yes, we should use `DUST_CLIENT_FACING_URL` according to [this doc](https://www.notion.so/dust-tt/DesignDoc-Dust-services-URLs-revamp-8e1aab69fdb84809b8be4fc9f75e719e).
It is already set for front, so you just  need to use this env.